drools-wb-webapp/pom.xml

```diff
@@ -1331,37 +1519,51 @@
 
             <!-- drools-wb -->
             <compileSourcesArtifact>org.drools:drools-wb-enum-editor-api</compileSourcesArtifact>
-            <compileSourcesArtifact>org.drools:drools-wb-enum-editor-client</compileSourcesArtifact>
+            <!-- Not needed - add as sources !-->
+            <!--<compileSourcesArtifact>org.drools:drools-wb-enum-editor-client</compileSourcesArtifact>-->
```

Comment:
Here too a lot of commented modules. These will get outdated at one point and since they are not tied to the compile they start to be misleading.